index.js

```diff
@@ -19,35 +19,40 @@ var appServer = function(config) {
     port: process.env.port || 8080,
     httpEnabled: true,
     httpsEnabled: false,
-    server_root: ''
+    public_html: 'public_html',
+    server_dir: 'server',
+    server_root: '.',
```

Comment:
I would recommend not defaulting `server_root` to `.`, rather `__dirname` would be safer. See http://stackoverflow.com/a/18283508 for more info.